terminus/generators/monolane_generator.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import print_function
+
+from collections import OrderedDict
+from math import atan2
+from math import degrees
+import yaml
+
+from .city_visitor import CityVisitor
+from models.intersection_node import IntersectionNode
+
+
+class MonolaneGenerator(CityVisitor):
```

Comment:
Ok, I refactored the `RNDFGenerator` test cases into a separate module in 6616c51 and used those to implement similar tests for the `MonolaneGenerator` in dcaa31a.